packages/bot-skeleton/src/services/tradeEngine/utils/helpers.js

```diff
@@ -52,15 +53,15 @@ const getBackoffDelayInMs = (error, delay_index) => {
     const max_delay = 15;
     const next_delay_in_seconds = Math.min(base_delay * delay_index, max_delay);
 
-    if (error?.name === 'RateLimit') {
+    if (error.error.code === 'RateLimit') {
```

Comment:
Why don't we need optional chaining anymore?

Reply:
on our code, some of our errors return as response object(error.error.name) the other errors return as error object(error.name) but on updating code I try to send all error in a similar way so we don't need this optional chaining